xmtp_mls/README.md

```diff
@@ -0,0 +1,172 @@
+# XMTP MLS
```

Comment:
Thank you for this README, @neekolas! What do you think of updating the title and adding a bit of intro text for context? Some suggested starter text here:

```suggestion
# XMTP group chat using MLS

This document describes plans for how XMTP can provide group chat using [Messaging Layer Security](https://messaginglayersecurity.rocks/) (MLS).
```

Reply:
The library today only handles group chat, but in future it's gong to include 1:1 as well. 

I will add the intro text, which helps set the reader up for the onslaught of technical detail that follows.